<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EleveSpWidget</class>
 <widget class="QWidget" name="EleveSpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EleveSpWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lNamePar">
     <property name="text">
      <string>Prenom parent:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leNamePar"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lEmailPar">
     <property name="text">
      <string>Email parent:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leEmailPar"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lTelPar">
     <property name="text">
      <string>Tel parent:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leTelPar"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
